src/Admin/Support.php

```diff
@@ -0,0 +1,187 @@
+<?php
+/**
+ * Class SupportMenu
+ *
+ * @package Ampproject\Ampwp
+ */
+
+namespace AmpProject\AmpWP\Admin;
+
+use AmpProject\AmpWP\Infrastructure\Conditional;
+use AmpProject\AmpWP\Infrastructure\Registerable;
+use AmpProject\AmpWP\Infrastructure\Service;
+use AmpProject\AmpWP\QueryVar;
+
+/**
+ * SupportMenu class.
+ */
```

Comment:
Make sure all classes are marked with `@internal`.

```suggestion
 * @internal
 */
```